schema/project-level/project-package-schema.json

```
@@ -91,7 +91,7 @@
     },
     "publicationPolicy": {
       "title": "Publication policy",
-      "description": "A link to a document describing the publisher's [publication policy](https://standard.open-contracting.org/latest/en/implementation/publication_policy/).",
+      "description": "A link to a document describing the publisher's [guidance for data users](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/guidance/guidance_for_data_users.md).",
```

Sphinx doesn't do it's clever URL magic on the schema, so URLs need to be the actual URL of the built page, rather than the raw markdown file:

```suggestion
      "description": "A link to a document describing the publisher's [guidance for data users](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/guidance/guidance_for_data_users/).",
```